L/libpolymake_julia/build_tarballs.jl

```diff
@@ -5,22 +5,21 @@ using Base.BinaryPlatforms
 
 
 name = "libpolymake_julia"
-version = v"0.5.0"
+version = v"0.6.0"
 
 julia_versions = [v"1.6.0", v"1.7.0", v"1.8.0"]
 
 # Collection of sources required to build libpolymake_julia
 sources = [
     ArchiveSource("https://github.com/oscar-system/libpolymake-julia/archive/v$(version).tar.gz",
-                  "5af8ccce9928b05c2c30f6adcddc7f07bdbef79d69d7d28aa6671846e742d2fb"),
+                  "de2180654322b2e68e47dd59bb842ccb7956e81bafa618dafef0320c35189968"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-# remove $libdir from LD_LIBRARY_PATH as this causes issues with perl
-if [[ -n "$LD_LIBRARY_PATH" ]]; then
-LD_LIBRARY_PATH=$(echo -n $LD_LIBRARY_PATH | sed -e "s|[:^]$libdir\w*|:|g")
-fi
+# change default perl which interferes with the hostbuild perl
+rm -f /usr/bin/perl
+ln -s $host_bindir/perl /usr/bin/perl
```

Comment:
Is this symlink really necessary after you delete the perl in /usr/bin? The host bindir should be in the PATH, if that's useful. Anyway, this isn't blocking for this PR

Reply:
For some reason the original `polymake-config` tool has [`/usr/bin/perl` hardcoded](https://github.com/polymake/polymake/blob/Releases/perl/polymake-config#L1). And in some intermediate version of `polymake_jll` this wasn't patched away, so it was really needed to run that tool. By now it should not be necessary anymore and I can remove it in the next iteration.